internal/service/advancedcluster/model_advanced_cluster.go

```diff
@@ -446,25 +455,29 @@ func expandBiConnectorConfig(d *schema.ResourceData) *admin.BiConnector {
 	return nil
 }
 
-func flattenProcessArgs(p *admin20240530.ClusterDescriptionProcessArgs) []map[string]any {
-	if p == nil {
+func flattenProcessArgs(p20240530 *admin20240530.ClusterDescriptionProcessArgs, p *admin.ClusterDescriptionProcessArgs20240805) []map[string]any {
+	if p20240530 == nil {
```

Comment:
names looks strange to me. I would go for a more generic: `pOld` and `pNew`

Reply:
I did it like this to be consistent on how we handle different SDK version and also thinking in the future where we might only use the "new", so the refactor would be cleaner and we would only keep the `p` (there wouldn't be a need to change from `pNew` to `p`)